deploy/crds/sriovnetwork.openshift.io_sriovnetworks_crd.yaml

```diff
@@ -74,6 +84,8 @@ spec:
               minimum: 0
               type: integer
           required:
+          - max_tx_rate
```

Comment:
I am not sure if this should be required? Same for min_tx_rate?

Reply:
maybe add `omitempty` in `MinTxRate` and `MaxTxRate` type definition, so it can be omitted if not specified.

Reply:
Added omitempty. Thanks.